src/extractor/edge_based_graph_factory.cpp

```diff
@@ -241,8 +247,7 @@ unsigned EdgeBasedGraphFactory::RenumberEdges()
             // oneway streets always require this self-loop. Other streets only if a u-turn plus
             // traversal
             // of the street takes longer than the loop
-            m_edge_based_node_weights.push_back(edge_data.distance +
-                                                profile_properties.u_turn_penalty);
+            m_edge_based_node_weights.push_back(edge_data.weight);
```

Comment:
@TheMarex question about this line here - why was the `u_turn_penalty` removed from the `edge_based_node_weights`? On the branch, this test [in guidance](https://github.com/Project-OSRM/osrm-backend/blob/master/features/guidance/turn-angles.feature#L1091-L1120) fails to create a route unless the extra 200 is added to the edge based node weights. I can't tell why this one line makes a difference, but I do know if I add an arbitrary `200` to each node weight again, that test passes again.